src/api.v2/model/CellLevelMeta.js

```diff
@@ -1,17 +1,21 @@
 const BasicModel = require('./BasicModel');
 const sqlClient = require('../../sql/sqlClient');
 const tableNames = require('./tableNames');
+const bucketNames = require('../../config/bucketNames');
+const { getSignedUrl } = require('../helpers/s3/signedUrl');
 
 const fields = [
   'id',
   'name',
   'upload_status',
   'created_at',
+  'size',
 ];
 
 class CellLevelMeta extends BasicModel {
   constructor(sql = sqlClient.get()) {
     super(sql, tableNames.CELL_LEVEL, fields);
+    this.bucketName = bucketNames.CELL_METADATA;
```

Comment:
If it is not too bothersome, please fix namings for `CELL_LEVEL` and `CELL_METADATA` to be consistent.